app/experimenter/templates/experiments/detail_base.html

```diff
@@ -193,18 +193,60 @@ <h5 class="col mt-3">History</h5>
       <div class="row">
         <div class="col">
           {{ user }}
-
           {% for user_change in user_changes %}
+            {% if "Edited Experiment" in user_change.pretty_status %}
+            <p id="editExperiment{{user_change.id}}" data-toggle="modal" data-target="#changelog-modal{{user_change.id}}">
+              <span class="fas fa-caret-right"></span>
+              Edited Experiment
+            </p>
+
+            <div id="changelog-modal{{user_change.id}}" class="modal" tabindex="-1" role="dialog">
+                <div class="modal-dialog modal-lg" role="document">
+                  <div class="modal-content">
+                    <div class="modal-header">
+                        <h5 class="modal-title"> {{user}} edited on {{user_change.changed_on}}</h5>
```

Comment:
I changed my mind, it's not as readable as I'd hoped, I think this is better:

                      <div class="col">
                        <h5 class="modal-title">Edited by {{user}} </h5>
                        <p class="text-muted">{{user_change.changed_on}}</p>
                      </div>


<img width="278" alt="Screenshot 2019-08-13 11 18 53" src="https://user-images.githubusercontent.com/119884/62953980-90a8e580-bdbc-11e9-9895-dd0b9ad3a7d4.png">